cli_flags.go

```diff
@@ -35,6 +37,24 @@ const (
 	maxArgMapScaleFactor = 8
 )
 
+type SymbolEndpoint struct {
+	Site   string
+	APIKey string
+	AppKey string
+}
```

Comment:
It's generally idiomatic to add struct field tags when parsing json, to keep attributes in snake case and lowercase:

```suggestion
type SymbolEndpoint struct {
	Site   string `json:"site"`
	APIKey string `json:"api_key"`
	AppKey string ...
}
```